packages/components/FlexCol.tsx

```diff
@@ -0,0 +1,39 @@
+import { FlexAlignType, View, ViewProps } from "react-native";
```

Comment:
In fact, we have discussed with @n0izn0iz  about this component, this add a very thin layer (```flexDirection: "column"```) on top of View so I hesitate to keep it, but at the same time, there is FlexRow (that Norman suggested to change the name to BrokenRow or sth like that). wdyt @n0izn0iz  ?

Reply:
We'll see later.
But I think using a component to handle flex rows/cols is good

Reply:
adding a level of indirection for one style field is really bad IMO